api/types/accesslist/review.go

```diff
@@ -39,6 +39,17 @@ type Review struct {
 	Spec ReviewSpec `json:"spec" yaml:"spec"`
 }
 
+// Clone returns a copy of the review.
+func (a *Review) Clone() *Review {
+	if a == nil {
```

Comment:
why not using goderive for everything?

Reply:
Let keep this discussion in https://github.com/gravitational/teleport/pull/58588